src/libs/Altinn.Authorization.Core.Models/src/Altinn.Authorization.Core.Models/Register/OrganizationNumber.cs

```diff
@@ -0,0 +1,133 @@
+﻿#nullable enable
+
+using System.Buffers;
+using System.Diagnostics.CodeAnalysis;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Altinn.Swashbuckle.Examples;
+
+namespace Altinn.Authorization.Core.Models.Register;
+
+/// <summary>
+/// A organization number (a string of 9 digits).
+/// </summary>
+[JsonConverter(typeof(JsonConverter))]
+public class OrganizationNumber : ISpanParsable<OrganizationNumber>,
```

Comment:
This should probably implement `IEquatable<OrganizationNumber>`, and overload the equality operators. See `PersonIdentifier`.